dasharo-security/secure-boot.robot

```diff
@@ -285,6 +285,58 @@ SBO009.002 Check automatic certificate provisioning KEK certificate
     ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
     Reset To Default Secure Boot Keys    ${advanced_menu}
 
+SBO010.001 Enroll certificates using sbctl
+    [Documentation]    This test installs sbctl and verifies that it is possible
+    ...    to use it to enroll certificates.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO010.001 not supported
+    Power On
+    # Erase Secure Boot Keys
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    Enable Secure Boot    ${sb_menu}
+    Save Changes
+    Reenter Menu
+    ${sb_menu}=    Get Secure Boot Menu Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # Install sbctl
+    Execute Linux Command    apt install -y git util-linux binutils golang-go asciidoc make    120
```

Comment:
@PLangowski it would be a lot better to prepare preconfigured Ubuntu via https://github.com/Dasharo/preseeds with sbctl installed

Reply:
From now on it is being installed using Ansible. Added here fc327b7c402d81c4523d763476be55025713e509, amended in following commits: fb722ce7e298b2e429cbaffd918498ea858f01eb, 160d1dd04b31c18bc38e1804c68f990bdf4e6112 and e7bbb51d2a28d22515a6b17eb11bd92e13d6129b.